### PR TITLE
added  check

### PR DESCRIPTION
**Pull request by ajstanley**
**Issue 1030*: ([link](https://github.com/Islandora/islandora/issues/1030))

# What does this Pull Request do?

If an Islandora installation does not use Fedora then viewing a media object will cause a fatal error

# What's new?
A check for $fedora_root has been added to islandora_entity_view

# How should this be tested?

Delete or comment out the flysystem config from settings.php, then try to view a media

# Additional Notes:
Any additional information that you think would be helpful when reviewing this
 PR.

# Interested parties
@Islandora/committers

### CODE REVIEW

**Comment by adam-vessey** [comment]
Shouldn't the display not be defining `field_gemini_uri`, if Fedora's not in use? https://github.com/Islandora/islandora/pull/1031/files#diff-9f1c9d46d88dce55fa1d0ce1103dd4184b824342407a63c2397844703c09b78eR548

**Comment by rosiel** [comment]
from Tech call: 
* We'll clean up this section of code... thanks @ajstanley and @adam-vessey 
* We'll also make a ticket for moving the "fedora stuff" to its own module

File: islandora.module [hunk 1]
```diff
@@ -545,12 +545,13 @@ function islandora_entity_view(array &$build, EntityInterface $entity, EntityVie
   $route_match_item = \Drupal::routeMatch()->getParameters()->get($entity->getEntityTypeId());
   // Ensure the entity matches the route.
   if ($entity === $route_match_item) {
-    if ($display->getComponent('field_gemini_uri')) {
+    $flysystem_config = Settings::get('flysystem');
+    $fedora_root = $flysystem_config['fedora']['config']['root'];
```
Comment (adam-vessey) [inline islandora.module]: This attempted access would likely start throwing warnings in environments where it isn't defined.

Comment (ajstanley) [reply islandora.module]: getComponent() returns NULL if the component is not defined.

**Review by rosiel — APPROVED** [review]

**Pull request merged**